modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/GPAD_conf.pm

```diff
@@ -241,9 +241,41 @@ sub pipeline_analyses {
                               delete_existing => $self->o('delete_existing'),
                               logic_name => $self->o('logic_name')
        },
+      -flow_into         => 'RunXrefCriticalDatacheck',
       -analysis_capacity  => 20,
       -rc_name 	      => 'default'
     },
+    {
+      -logic_name        => 'RunXrefCriticalDatacheck',
+      -module            => 'Bio::EnsEMBL::DataCheck::Pipeline::RunDataChecks',
+      -parameters        => {
+                              datacheck_groups => ['xref'],
+                              datacheck_types  => ['critical'],
+                              registry_file    => $self->o('registry'),
+                              history_file    => $self->o('history_file'),
+                              failures_fatal  => 1,
+                            },
+      -flow_into         => 'RunXrefAdvisoryDatacheck', 
+      -max_retry_count   => 1,
+      -analysis_capacity => 10,
+      -batch_size        => 10,
+    },
+    {
+      -logic_name        => 'RunXrefAdvisoryDatacheck',
```

Comment:
Advisory checks often compare data to the previous release's database, so a parameter is required to specify the appropriate server. One way to do this is with `-old_server_uri`, and then have the Confluence SOP pass in the mirror server that is appropriate for each staging server.
But a cleaner way to do it is to use the shared config files in the datacheck/config directory. There's one per staging server, and they already contain the necessary old server details. They also contain standard locations for history and output too, so it's three birds with one stone. For maximum flexibility in calling the pipeline, probably best for config_file to have a default of undef, as with history_file.